resources/js/app.js

```diff
@@ -233,6 +234,22 @@ const router = new VueRouter({
             name: 'events',
             component: EventList
         },
+
+        {
+            path: '/chat',
+            name: 'chat',
+            component: ChatComponent
+        },
+
+        {
+            path: 'messages',
+            name: 'messages'
+        },
+
+        {
+            path: 'send-message',
+            name: 'send-message'
+        },
```

Comment:
Nie wiem czy to jest nam potrzebne

Reply:
pierwsze jest do wyświetlenia wiadomości w czasie, a to drugie to wysyłania wiadomości